main/getting-started/beta.md

```diff
@@ -21,7 +21,7 @@ Don't worry, this'll get much easier soon. After it passes reviews, the wallet w
 be available via a downloadable application. On mainnet launch, Agoric will be integrated 
 with leading wallet providers via a simple plugin.
 
-## Head to the Treasury ##
+## Head to the Treasury App ##
 
 Play around with the mock assets we've kindly pre-stocked your wallet with.
```

Comment:
Line 28: 

`[Treasury Application](https://treasury.agoric.app)`

should be

`[Vault Application](https://treasury.agoric.app)`

(The link only works as treasury.agoric.app currently.)

Reply:
okay no worries I figured we could just continue to call it the "Treasury App" until we got the URL changed. I'll fix it! 😊